ert3/storage/__init__.py

```diff
@@ -0,0 +1,138 @@
+import ert3
```

Comment:
Why is everything put in `__init__.py` files? Is it necessary to import everything always?